packages/dd-trace/src/constants.js

```diff
@@ -6,5 +6,7 @@ module.exports = {
   ANALYTICS_KEY: '_dd1.sr.eausr',
   ORIGIN_KEY: '_dd.origin',
   HOSTNAME_KEY: '_dd.hostname',
-  REFERENCE_NOOP: 'noop'
+  REFERENCE_NOOP: 'noop',
+  LOG_EXPORTER: 'log-exporter',
```

Comment:
These are not needed. While this is not clear at a glance, this file is only for private constants that are not exposed externally.